### PR TITLE
add package: feathernotes

### PR DESCRIPTION
**Pull request by Yisus7u7**
Screenshot_20210728-180343

### CODE REVIEW

**Comment by Yisus7u7** [comment]
@xeffyr

```
/home/builder/.termux-build/feathernotes/src/feathernotes/fn.cpp:2907:43: error: no member named 'families' in 'QFont'
        defaultFont_.setFamilies (newFont.families()); // to override the existing font families
                                  ~~~~~~~ ^
/home/builder/.termux-build/feathernotes/src/feathernotes/fn.cpp:2907:22: error: no member named 'setFamilies' in 'QFont'; did you mean 'setFamily'?
        defaultFont_.setFamilies (newFont.families()); // to override the existing font families
                     ^~~~~~~~~~~
                     setFamily
/data/data/com.termux/files/usr/include/QtGui/qfont.h:180:10: note: 'setFamily' declared here
    void setFamily(const QString &);
         ^
2 errors generated.
[40/52] Building CXX object feathernotes/CMakeFiles/feathernotes.dir/find.cpp.o
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1. ```

It's a very rare mistake, why?

**Comment by xeffyr** [comment]
Maybe our Qt is just too old, we use `5.12` where 5.15 is out...

**Comment by Yisus7u7** [comment]
> Maybe our Qt is just too old, we use `5.12` where 5.15 is out...

Then I'll look forward to the qt update

**Comment by Yisus7u7** [comment]
I will use the direct version of git, this will be a temporary solution while termux updates Qt

File: packages/feathernotes/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tsujan/FeatherNotes
+TERMUX_PKG_DESCRIPTION="Lightweight Qt Notes-Manager for Linux"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Yisus7u7"
+TERMUX_PKG_VERSION=0.9.0
+TERMUX_PKG_SRCURL=https://github.com/Yisus7u7/shpkg-builds/releases/download/${TERMUX_PKG_VERSION}/feathernotes-${TERMUX_PKG_VERSION}.tar.xz
```
Comment (xeffyr) [inline packages/feathernotes/build.sh:6]: Custom sources aren't allowed. If you have made any changes to the source code, bundle patches within directory where build.sh is stored and set TERMUX_PKG_SRCURL to the original upstream.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by xeffyr** [comment]
> I will use the direct version of git,

If it works with the latest git version, then issue is not "outdated Qt".

**Comment by Yisus7u7** [comment]
> > Usaré la versión directa de git,
> 
> Si funciona con la última versión de git, entonces el problema no es "Qt desactualizado".

Compile feathernotes directly Clone the git repository, it works, but using version release doesn't work

**Comment by Yisus7u7** [comment]
I didn't and added patches, they wouldn't do them, nor did they modify the source code, just the direct version of git

**Comment by xeffyr** [comment]
**Use original URL!**

You don't need to manually fetch and store git sources somewhere else. TERMUX_PKG_SRCURL may contain URL to the repository managed by original author or fork managed by community.

**Comment by Yisus7u7** [comment]
Then it's time to patch, compare the git code with the one in the releases, how do you make a patch?

**Comment by xeffyr** [comment]
https://github.com/tsujan/FeatherNotes/archive/087f522d62deed421b40d8fe96d66ecef0f38bf3.zip

...

**Comment by Yisus7u7** [comment]
> https://github.com/tsujan/FeatherNotes/archive/087f522d62deed421b40d8fe96d66ecef0f38bf3.zip
> 
> ...

That's the last commit  to the git repository?

**Comment by xeffyr** [comment]
Yes, it is: https://github.com/tsujan/FeatherNotes/commit/087f522d62deed421b40d8fe96d66ecef0f38bf3. Latest commit on master branch.

**Comment by xeffyr** [comment]
I have asked you to change only TERMUX_PKG_SRCURL. Does that mean to change version as well?

***

See an example of how it should look like:
https://github.com/termux/termux-packages/blob/79ed512695309db6d4bc2bec15ce9cc281ab7f66/packages/proot/build.sh#L6-L10

**Review by xeffyr — APPROVED** [review]

**Pull request merged**